importScripts/sampleRecords/locationHistory.destination.json

```diff
@@ -164,38 +164,6 @@
     {
       "movement_id":"11",
       "id":"11",
-      "idno":"108",
```

Comment:
Part of the data cleaning resulted storage locations being marked as old (they no longer exist), and we therefore ignore them on import.